docs/interfaces/authenticatedroute.md

```diff
@@ -25,7 +25,7 @@
 
 • **isAuthenticated**: *boolean*
 
-*Defined in [src/interfaces/authenticated-route.ts:12](https://github.com/brandongregoryscott/AndcultureCode.JavaScript.React/blob/b4a48e2/src/interfaces/authenticated-route.ts#L12)*
+*Defined in [src/interfaces/authenticated-route.ts:12](https://github.com/wintondeshong/AndcultureCode.JavaScript.React-1/blob/3bed964/src/interfaces/authenticated-route.ts#L12)*
```

Comment:
These docs are great, but I'm noticing these link to the forked repo.  Is there a way to link to the andculture repo?